app/assets/javascripts/listings/templates/listing/_what_to_expect.html.slim

```diff
@@ -1,18 +1,17 @@
 h4.h-caps-lined
   | What to Expect
 p.c-steel.t-small.margin-bottom
-  ' Applicants will be contacted by the {{$ctrl.parent.isSale($ctrl.parent.listing) ? 'sales agent' : 'leasing agent'}}
+  ' Applicants will be contacted by the property agent
   span ng-if="!$ctrl.parent.isFirstComeFirstServe($ctrl.parent.listing)"
     | in lottery rank order until vacancies are filled.
   span ng-if="$ctrl.parent.isFirstComeFirstServe($ctrl.parent.listing)"
     | on a first-come, first-serve basis until the vacancy is filled.
 .show-for-print ng-show="$ctrl.parent.toggleStates[$ctrl.parent.listing.Id].whatToExpectOpen"
-  p.c-steel.t-small
-    | All of the information that you have provided must be verified, and your eligibility confirmed.  Any fraudulent statements will cause your application to be removed from the lottery.  You will also be removed from the lottery if you cannot prove a housing preference, or if a household member appears on more than one application.
+  p.c-steel.t-small translate="F2_REVIEW_TERMS.P3"
```

Comment:
What are our thoughts on re-using text from application stuff elsewhere? I can't decide if it's worse to have the same language copy and pasted in two places, or if it's worse to have someone update the content assuming it's just for the application then have it have unforeseen repercussions.

Reply:
if it's the same piece of text conceptually, then it makes sense to use the same translation term. i.e., if the intention is that we should use the same block of text to convey the concept of "information veracity requirements" on both the listing and the application, then we should use the same translation term. but if the two pieces of text are not meant to be the same conceptually, then it would be better to use separate terms. could product/design help clarify the intention in this particular case?

Reply:
Ok. checked with Emily. I'm going to copy and paste because 3rd paragraph of Review terms doesn't communicate the concept so there's a high risk of unintended consequences.